src/content/basics/aws-architecture/index.md

```diff
@@ -13,33 +13,33 @@ Giant Swarm's Architecture is split into two logical parts, one being the Contro
 
 ## Giant Swarm control plane
 
-![Control Plane Architecture](/img/architecture-aws-control-plane.png)
+![Control Plane Architecture](architecture-aws-control-plane.png)
 
 The above diagram shows our control plane setup running over three availability zones. We setup two bastion hosts in different availability zones. All instances are in a private subnet and only accessible through the bastion hosts. Bastions hosts are available via AWS VPN connection only.
 
-The cluster has several APIs and Interfaces. The Kubernetes API of the Control Plane is only available to Giant Swarm operations personnel and only through the Bastion hosts. 
+The cluster has several APIs and Interfaces. The Kubernetes API of the Control Plane is only available to Giant Swarm operations personnel and only through the Bastion hosts.
 
 The Giant Swarm API, Monitoring and Alerting frontends as well as our Web Management Interface, Happa, are exposed through the Control Plane Ingress Controller, which sits behind a public ELB.
 
 Access to those interfaces can be configured to work with the customers' Identity Management System.
 
 ## Giant Swarm tenant cluster
 
-![Tenant Cluster Architecture](/img/architecture-aws-tenant-cluster.png)
+![Tenant Cluster Architecture](architecture-aws-tenant-cluster.png)
 
-Via the Giant Swarm API, our [CLI](https://github.com/giantswarm/gsctl), or our Happa interface, you can start Tenant Clusters of different sizes. There's a selection of recommended EC2 instance types, which can be adjusted if needed. 
+Via the Giant Swarm API, our [CLI](https://github.com/giantswarm/gsctl), or our Happa interface, you can start Tenant Clusters of different sizes. There's a selection of recommended EC2 instance types, which can be adjusted if needed.
```

Comment:
I'd like us to start eliminating Happa from documentation and use "our web UI" instead.

Reply:
changed everywhere to  "our Web User Interface"